src/overlays/actors/ovl_En_Ssh/z_en_ssh.c

```diff
@@ -419,7 +419,7 @@ void EnSsh_Sway(EnSsh* this) {
         }
 
         temp_f20 = (this->swayTimer * (1.0f / 6));
-        swayAngle = DEG_TO_BINANG_ALT3(temp_f20) * Math_SinS(this->swayAngle);
+        swayAngle = (f32)DEG_TO_BINANG_ALT3(temp_f20) * Math_SinS(this->swayAngle);
```

Comment:
Why the explicit cast here?

Reply:
Main reason is it feels like without it, the macro kind of hides what the type actually is. Generally you would expect `s16` for binangs, so I just like making it explicit. It also matches all the other usages that require explicit `(s32)` casts.